src/subdevice/mod.rs

```diff
@@ -326,6 +326,11 @@ where
     pub fn set_dc_sync(&mut self, dc_sync: DcSync) {
         self.state.dc_sync = dc_sync;
     }
+
+    /// TODO
+    pub fn as_raw_ref_mut(&mut self) -> SubDeviceRef<'a, &mut SubDevice> {
+        SubDeviceRef { maindevice: self.maindevice, configured_address: self.configured_address, state: self.state.deref_mut() }
+    }
```

Comment:
This is maybe a very me-specific problem but I wanted to pass the SubDeviceRefs to a trait object function, so I needed a concrete type.